pyangbind/lib/serialise.py

```diff
@@ -110,7 +117,7 @@ def default(self, obj, mode='default'):
         return unicode(obj)
     elif orig_yangt in ["int8", "int16", "int32", "uint8", "uint16", "uint32"]:
       return int(obj)
-    elif orig_yangt in ["int64" "uint64"]:
+    elif orig_yangt in ["int64", "uint64", "oc-types:timeticks64", "oc-yang:counter64"]:
```

Comment:
This doesn't seem robust, can we do something here for where we have a typedef to resolve it into the fact that it's really based on one of the 64-bit integer types?

Reply:
This turns out to be the same issue as above, where we're having to add typedefs into the library code. We should strive to avoid this, otherwise the library isn't portable to any YANG module.